.gitlab-ci.yml

```diff
@@ -276,20 +136,19 @@ trigger_pipeline:
         echo "Can not find the corresponding Pull Request"
       fi
   # Override variables condition from .pr_condition
-  only:
-    variables:
-      - $RUN_CI_TAG && $STATUS_CONTEXT == "quick"
+  rules:
+    - if: $RUN_CI_TAG && $STATUS_CONTEXT == "quick"
```

Comment:
the rules override the pr condition rules. https://docs.gitlab.com/ee/ci/yaml/yaml_optimization.html#merge-details
it will run the trigger_pipline in master, develop, tags now

Reply:
good point! should be fixed